src/Avalonia.Controls/ToolTipService.cs

```diff
@@ -97,6 +99,17 @@ private void ControlPointerLeave(object sender, PointerEventArgs e)
             Close(control);
         }
 
+        private void Control_PropertyChanged(object sender, AvaloniaPropertyChangedEventArgs e)
+        {
+            var control = (Control)sender;
+
+            if (e.Property == Visual.BoundsProperty)
```

Comment:
Note: BoundsProperty has relative position value and ToolTip position will not be recalculated in some cases, when absolute position was actually changed. For example, scrolling.
It is possible to use TransformedBoundsProperty, but it's not updated when control is not rendered, and might be discontinued in the future.

Reply:
Replaced with better alternative - EffectiveViewportChanged